Add portsdb.adoc

yurivict opened the pull request.

pauamma commented on 2022q4/portsdb.adoc:
```diff
@@ -0,0 +1,37 @@
+Links: +
+link:https://github.com/yurivict/freebsd-portsdb/[PortsDB] URL: link:https://github.com/yurivict/freebsd-portsdb/[https://github.com/yurivict/freebsd-portsdb/]
+
+
```
Spurious line break.

pauamma commented on 2022q4/portsdb.adoc:
```diff
@@ -0,0 +1,37 @@
+I developed the PortsDB project that imports FreeBSD ports into an SQLite
+database. The port is ports-mgmt/portsdb.
+
+The database can be fully rebuilt in around 20 minutes, after which in can
```
s/in can/it can/

pauamma commented on 2022q4/portsdb.adoc:
```diff
@@ -0,0 +1,37 @@
+https://people.freebsd.org/~yuri/ports.sqlite
+
+PortsDB can be used to query ports using SQL, as a relational database.
+External services like Repology, FreshPorts, Portscout and similar can use
```
s/Portscout/Portscout,/ (serial comma)

pauamma commented on 2022q4/portsdb.adoc:
```diff
@@ -0,0 +1,37 @@
+All that needs to be done to run PortsDB is ./import.sh and then ./update.sh
+
+One particular immediate problem that PortsDB aims to solve is to fix
+inaccuracy in how FreeBSD port versions are displayed by Repology.
```
s/inaccuracy/inaccuracies/ (or "an inaccuracy")

pauamma commented on 2022q4/portsdb.adoc:
```diff
@@ -0,0 +1,37 @@
+inaccuracy in how FreeBSD port versions are displayed by Repology.
+Repology uses ports INDEX which is missing some required information.
+This leads to Repology not being able to distinguish between real
+versions and intermediate and made up versions. PortsDB should allow
```
s/made up/made-up/

pauamma reviewed (commented):
Cosmetic changes. I can apply them while/after merging if you agree.

yurivict commented:
Hi @pauamma ,

I've updated the PR. I fixed most issues that you mentioned, and also reworded some places and added more content.

Please feel free to make additional corrections whole merging.

(Particularly I didn't understand the spurious line break issue.)

pauamma reviewed (approved):
Will do. (By "spurious", I meant there are 2 where 1 is enough.)
